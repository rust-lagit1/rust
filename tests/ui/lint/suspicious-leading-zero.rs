#![deny(suspicious_leading_zero)]

fn main() {
    let _ = 0111; //~ ERROR [suspicious_leading_zero]
    let _ = 0007; //~ ERROR [suspicious_leading_zero]
    let _ = 0008;
}
