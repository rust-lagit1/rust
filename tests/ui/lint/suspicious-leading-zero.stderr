error: TODO
  --> $DIR/suspicious-leading-zero.rs:4:13
   |
LL |     let _ = 0111;
   |             ^
   |
note: the lint level is defined here
  --> $DIR/suspicious-leading-zero.rs:1:9
   |
LL | #![deny(suspicious_leading_zero)]
   |         ^^^^^^^^^^^^^^^^^^^^^^^
help: TODO
   |
LL |     let _ = 0o111;
   |             ~~

error: TODO
  --> $DIR/suspicious-leading-zero.rs:5:13
   |
LL |     let _ = 0007;
   |             ^
   |
help: TODO
   |
LL |     let _ = 0o007;
   |             ~~

error: aborting due to 2 previous errors

